.github/workflows/test.yml

```
@@ -0,0 +1,59 @@
+name: Test
+
+on:
+  # Run each time we push and pull requests
+  push:
+  pull_request:
+  # Cron job
+  # https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#scheduled-events-schedule
+  schedule:
+    # https://crontab.guru/#0_0_1_*_*
+    - cron: "0 0 1 * *"
+
+jobs:
+  # As we are running on different environments, we are splitting the jobs
+  # https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobs
+  local:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [3.7, 3.8]
+        os: [ubuntu-latest]
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Setup prerequisites
+        run: |
+          sudo apt update
+          sudo apt install libhdf5-serial-dev
+          sudo apt install netcdf-bin
+          sudo apt install libnetcdf-dev
+          sudo apt install libsm6
+          sudo apt install libxext6
+          sudo apt install libxrender-dev
+          sudo apt install libxt6
+          sudo apt install libgl1-mesa-glx
+          sudo apt install libfontconfig
+          sudo apt install libxkbcommon-x11-0
```

I usually like to include the `-y` flag so we don't accidentally get stuck on something asking us to accept a licence.

Also, this is the possibly a nicer way to avoid multiple calls to `apt`, relying on YAML seeing it as a single line string:

```suggestion
        run: sudo apt update &&
             sudo apt install -y libhdf5-serial-dev
                netcdf-bin
                libnetcdf-dev
                libsm6
                libxext6
                libxrender-dev
                libxt6
                libgl1-mesa-glx
                libfontconfig
                libxkbcommon-x11-0
```

(YAML is particularly awful about multiline strings)